lib/inject_method.rb

```diff
@@ -0,0 +1,14 @@
+class My_Array < Array
+
+	def challenge first 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.